src/Route.js

```diff
@@ -0,0 +1,19 @@
+import { parseRoute } from "./parseRoute"
+
+export function Route(props) {
+  var location = props.location || window.location
+  var path = props.path
+  var match =
+    !path ||
+    parseRoute(path, location.pathname, {
+      exact: !props.parent
```

Comment:
Is the idea here that anyone who wants non-exact route matching will use `parent` even if the component being routed to has no child routes?

Reply:
@leavehouse Yes. Whether the component declares child routes or not is left to the user.